src/Sanderling/Sanderling.Exe/App.Motor.cs

```diff
@@ -46,11 +46,6 @@ WindowMotor GetMotor()
 			{
 				var Process = System.Diagnostics.Process.GetProcessById(EveOnlineClientProcessId.Value);
 
-				if (null == Process)
-				{
-					return null;
-				}
-
```

Comment:
Why remove this branch?
What about the case of `Process` being null? How else would this case be taken care of?